Framework/PythonInterface/plugins/algorithms/ComputeCalibrationCoefVan.py

```diff
@@ -54,9 +54,16 @@ def PyInit(self):
                              direction=Direction.Output),
                              ("Name the workspace that will contain the " +
                               "calibration coefficients"))
+        self.declareProperty("Temperature",
+                             defaultValue=Property.EMPTY_DBL,
+                             validator=FloatBoundedValidator(lower=0.0),
+                             direction=Direction.Input,
+                             doc=("Temperature during the experiment."))
         return
 
     def validateInputs(self):
+        """ Validate the inputs
+        """
         issues = dict()
         inws = self.getProperty("VanadiumWorkspace").value
         run = inws.getRun()
```

Comment:
I've tried again and can't reproduce the crashes, just add the unit into the docs on line 61 and it will be good to go.